include/envoy/common/platform.h

```diff
@@ -19,6 +19,9 @@
 #include <mswsock.h>
 #include <ws2tcpip.h>
 #include <mstcpip.h>
+#include <winsvc.h>
+#include <processenv.h>
+#include <shellapi.h>
```

Comment:
These hurt every source module compilation, for only a couple of source files. Let's please move them to service_base.h/.cc as appropriate.

Reply:
fixed in 37a4d683293d4aa7576fee9860d21b2229359d73